src/components/TodoDetails.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div>
+    <page-header></page-header>
+    <p> {{ getItems[index].name }} </p>
+    <home-button></home-button>
+  </div>
+</template>
+
+<script>
+  import { Vue, Component, Prop} from 'vue-property-decorator';
+  import {mapGetters} from 'vuex';
+  import PageHeader from './PageHeader.vue';
+  import HomeButton from './HomeButton';
+
+  @Component({
+    components: {
+      pageHeader: PageHeader,
+      homeButton: HomeButton
+    },
+    computed: {
+        ...mapGetters([
+            'getItems'
+        ])
+    }    
+  })
+
+  export default class TodoDetails extends Vue {
+    @Prop() index
+  }
+</script>
+
+<style scoped>
+  p {
```

Comment:
Try using classes or id-s instead of just referncing the HTML element on its own. Also the BEM methodology is a good option when giving class names, we haven't talked about it yet, but it worths a look into it.